.github/workflows/build_and_test.yml

```diff
@@ -410,10 +410,7 @@ jobs:
         shell: bash -euxo pipefail {0}
 
   trigger-e2e-tests:
-    runs-on: dev
-    container:
-      image: 369495373322.dkr.ecr.eu-central-1.amazonaws.com/rust:pinned
-      options: --init
+    runs-on: 'ubuntu-latest'
```

Comment:
There is `369495373322.dkr.ecr.eu-central-1.amazonaws.com/base:latest` image which might be preferable if the current policy is "use our runners everywhere except for mac os compile job". @zoete wdyt?

Reply:
The rust one might be a bit overkill for this purpose, the most recent `base:pinned` image has curl

Reply:
Is there such a thing as `base:pinned`? Have not found that, so replaced with `base:latest`, as suggested.

Reply:
Yes there is as you can see here: https://eu-central-1.console.aws.amazon.com/ecr/repositories/private/369495373322/base?region=eu-central-1